backend/de.metas.contracts/src/main/java/de/metas/contracts/modular/workpackage/impl/SalesInvoiceLineLogHandler.java

```diff
@@ -123,6 +126,15 @@ public BooleanWithReason doesRecordStateRequireLogCreation(@NonNull final I_C_In
 		final String description = TranslatableStrings.adMessage(MSG_ON_COMPLETE_DESCRIPTION, productName, qtyEntered)
 				.translate(Language.getBaseAD_Language());
 
+		final ProductPrice priceActual = Optional.of(invoiceLine)
```

Comment:
same approach as described above,
but in this case pls introduce IInvoiceBL.getPriceActual(I_C_InvoiceLine)